### PR TITLE
🔀🌐 [cs_CZ] add/update translations for vehicle, buildings, etc.

### PR DESCRIPTION
**Pull request by Tadjinton**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* cs_CZ

**What is included in your update?**
* fix src/i18n/cs_CZ.ts to real data from operacni-stredisko.cz + cs_CZ translation
* fix vehicles in src/i18n/cs_CZ/vehicles.ts to real data from operacni-stredisko.cz + cs_CZ translation
* fix buildings in src/i18n/cs_CZ/buildings.ts to real data from operacni-stredisko.cz + cs_CZ translation
* add schoolings to src/i18n/cs_CZ/schoolings.ts
* updated translations for src/i18n/cs_CZ/global.ts

### CODE REVIEW

File: src/i18n/cs_CZ/buildings.ts [hunk 1]
```diff
@@ -585,19 +663,19 @@ export default {
...
         caption: 'Pyrotechnická služba PČR',
         color: '#663300',
-        coins: 50,
-        credits: 5_000_000,
+        coins: 35,
+        credits: 200_000,
         levelPrices: {
-            credits: [10_000, 50_000, ...Array(18).fill(100_000)],
-            coins: [10, 15, ...Array(18).fill(20)],
+            credits: [10_000, 50_000, ...Array(3).fill(100_000)],
```
Comment (Cr4zyc4k3) [inline src/i18n/cs_CZ/buildings.ts:669]: This building has only 3 levels?

File: src/i18n/cs_CZ/buildings.ts [hunk 2]
```diff
@@ -585,19 +663,19 @@ export default {
...
         extensions: [],
-        levelcost: ['1. 10.000', '2. 50.000', '3.-20. 100.000'],
-        maxBuildings: 'No limit',
+        levelcost: ['1. 10.000', '2. 50.000', '3.-5. 100.000'],
+        maxBuildings: 'Bez limitu',
         maxLevel: 5,
         special: '',
         startPersonnel: 4,
-        startVehicles: ['Vozidlo pyrotechnika PČR'],
+        startVehicles: [],
```
Comment (Cr4zyc4k3) [inline src/i18n/cs_CZ/buildings.ts:678]: Building without start vehicle? That is, at least in the german version, unusual

File: src/i18n/cs_CZ/vehicles.ts [hunk 3]
```diff
@@ -214,104 +292,175 @@ export default {
...
     },
     23: {
         caption: 'SUV VZS ČČK',
         color: '#990000',
         credits: 10_000,
         coins: 25,
-        staff: { min: 1, max: 4 },
+        staff: {
+            min: 1, 
+            max: 4
```
Comment (jxn-30) [inline src/i18n/cs_CZ/vehicles.ts:356]: missing `,` here :upside_down_face:

File: src/i18n/cs_CZ/vehicles.ts [hunk 4]
```diff
@@ -47,16 +55,16 @@ export default {
...
-        possibleBuildings: [],
+        possibleBuildings: [0, 18],
     },
     5: {
         caption: 'RZP',
         color: '#9c1c1c',
         credits: 5000,
         coins: 25,
-        staff: { min: 3, max: 3 },
+        staff: { min: 2, max: 2 },
```
Comment (Cr4zyc4k3) [inline src/i18n/cs_CZ/vehicles.ts:65]: This looks curious for me as well.

**Review by Cr4zyc4k3 — COMMENTED** [review]
A few questions, may be the cz game loca, but looks good overall.

File: src/i18n/cs_CZ/buildings.ts (hunk 1 above)
Comment (Suchty112) [reply src/i18n/cs_CZ/buildings.ts:669]: that's right. it only has 5 in total, the first costs 10k, the second 50k and then it goes on 3x with 100k :)

File: src/i18n/cs_CZ/buildings.ts (hunk 2 above)
Comment (Suchty112) [reply src/i18n/cs_CZ/buildings.ts:678]: yep no start vehicle ^^

File: src/i18n/cs_CZ/vehicles.ts (hunk 4 above)
Comment (Suchty112) [reply src/i18n/cs_CZ/vehicles.ts:65]: image
nop and again this is right :D

File: src/i18n/cs_CZ/vehicles.ts (hunk 3 above)
Comment (Tadjinton) [reply src/i18n/cs_CZ/vehicles.ts]: Thank you for highlighting the errors. 😊

**Review by Cr4zyc4k3 — APPROVED** [review]

**Comment by Tadjinton** [comment]
Does anyone know where the mistake might have happened?

**Comment by jxn-30** [comment]
just a failure that unfortunately happens on macOS soemtimes when building the docs. I've restarted the failed test build :)

**Comment by Tadjinton** [comment]
Oh, I see. I was worried I had another error in my code.

**Pull request merged**